### PR TITLE
Dev feature

### PR DESCRIPTION
**Pull request by Mar12358**
### Added Features
- Documentation ( Readme file )
- Set up a new project with webpack that is based on the previous Webpack exercise.
- Create an index.html file with HTML markup. Create an empty To Do List placeholder (<div> element). 
- index.html file set as a template using the HTML Webpack Plugin.
- Create array of to do tasks (array of objects). Each task object contains three keys:
   - description [string].
   - completed [bool].
   - index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list appears in order of the index values for each task.
- List looks like a clone of the part of the minimalist project. (Using CSS)
- All source files (index.html, index.js, style.css, and images) are located in /src.

### CODE REVIEW

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Mar12358 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+input.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    e.preventDefault();
+    const task = Task.create(input.value, arryOfTasks.length);
+    arryOfTasks.push(task);
+    input.value = '';
+    localStorage.setItem('tasks', JSON.stringify(arryOfTasks));
+  }
+});
+
```
Comment (ghost) [inline src/index.js]: - [ ] Great work up to this point but there are some crucial aspects you have not met yet.
  - Currently, when I add a task, I have to refresh a page to see it. Kindly follow  this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) to guide you on the implementation
  - Kindly note that the property index should be set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
  
image

   - I was not able to edit and delete the tasks. Kindly implement these functionalities as they are very crucial to the application

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Mar12358 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Documentation


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+input.addEventListener('keydown', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    e.preventDefault();
+    const task = Task.create(input.value, arryOfTasks.length);
+    arryOfTasks.push(task);
+    input.value = '';
+    localStorage.setItem('tasks', JSON.stringify(arryOfTasks));
+  }
+});
```
Comment (AbelG101) [inline src/index.js]: - [ ] I really appreciate your effort in adding the "add" functionality but this milestone doesn't require you to implement that you just need to create a temporary array of todo task objects and render them by iterating.
- Requirement:
  image
- [ ] Please revisit the [requirements here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Mar12358 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED ✔️ 🟢 

## Hi @Mar12358 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbelG101** [comment]
- [ ] Hello Ezequiel, sorry for the late comment here but please make sure the PR title is descriptive and matches the current milestone.
- So instead of saying "Dev feature" you can rename to "To do list: Structure"

**Pull request merged**